src/handler/http/request/organization/org.rs

```diff
@@ -346,14 +328,137 @@ async fn create_user_rumtoken(
 )]
 #[post("/organizations")]
```

Comment:
Doesn't look like there's any `params` for this route. I'd remove that `params` from the utoipa doc.

Reply:
well, seems like an existing problem unrelated to this pr.